DigitalLearningSolutions.Web/Controllers/ForgotPasswordController.cs

```diff
@@ -16,19 +16,22 @@ public ForgotPasswordController(IPasswordResetService passwordResetService)
         }
 
         [HttpGet]
-        public IActionResult Index(ForgotPasswordViewModel? model)
+        public IActionResult Index()
         {
-            model ??= new ForgotPasswordViewModel();
-            return View(model);
+            if (User.Identity.IsAuthenticated)
```

Comment:
Can the User or Identity be null (I remember us handling nulls in LogoViewComponent previously)?

Reply:
VS seems to be fine with this, no warnings are present to say it could be null. I suspect this is because it is within a controller action rather than a ViewComponent as LogoViewComponent was, but I might need to do some more reading to confirm that as the exact reason. Either way, its probably safe as is.